sdk/src/main/java/com/microsoft/hydralab/performance/PerformanceInspectionResult.java

```diff
@@ -12,12 +14,13 @@ public class PerformanceInspectionResult {
     @SuppressWarnings("visibilitymodifier")
     public PerformanceInspection inspection;
     @SuppressWarnings("visibilitymodifier")
+    @Nullable
     public File rawResultFile;
     @SuppressWarnings("visibilitymodifier")
     // TODO: restrict the size of it.
     public Object parsedData;
 
-    public PerformanceInspectionResult(File rawResultFile, PerformanceInspection inspection) {
+    public PerformanceInspectionResult(@Nullable File rawResultFile, PerformanceInspection inspection) {
```

Comment:
@taoran6 Do we have a spec here that which one can be null or non-null?

Reply:
Currently, we don't have a spec yet.